sdk/eventhub/event-hubs/test/sender.spec.ts

```diff
@@ -198,6 +198,77 @@ describe("EventHub Sender #RunnableInBrowser", function(): void {
     });
   });
 
+  describe("Create batch", function(): void {
+    it("should be sent successfully", async function(): Promise<void> {
+      const producer = client.createProducer({ partitionId: "0" });
+      const totalMessages = 3;
+      const eventDataBatch = await producer.createBatch();
+      for (let i = 1; i <= totalMessages; i++) {
+        const messageStatus = eventDataBatch.tryAdd({ body: `${Buffer.from("Z".repeat(650000))}` });
+        if (!messageStatus) {
+          break;
+        }
+      }
+      debug(`${eventDataBatch.events.length} messages sent in total.`);
+      should.equal(
+        eventDataBatch.events.length !== totalMessages,
+        true,
+        "Client was not able to send any messages"
+      );
+      await producer.send(eventDataBatch);
+    });
+
+    it("with partition key should be sent successfully.", async function(): Promise<void> {
+      const producer = client.createProducer({ partitionId: "0" });
+      const eventDataBatch = await producer.createBatch({ partitionKey: 1 as any });
```

Comment:
Why not specify the `partitionKey` as a string?

Reply:
It can be number(for javascript users) or string. I just added that for the testing purpose.